config/rbac/role.yaml

```diff
@@ -6,6 +6,14 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
can we add helm values for operator CRDs?  Asking for read privilege for all resources is too demanding